manifests/m/Microsoft/AzureCLI/2.52.0/Microsoft.AzureCLI.installer.yaml

```diff
@@ -1,12 +1,17 @@
 # Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
 
 PackageIdentifier: Microsoft.AzureCLI
 PackageVersion: 2.52.0
+ReleaseDate: 2023-09-05
 Installers:
-- Architecture: x64
+- Architecture: x86
   InstallerType: msi
   InstallerUrl: https://azcliprod.azureedge.net/msi/azure-cli-2.52.0.msi
   InstallerSha256: b410e4fbd8602d46df54c77644209887c9f6bf87df08f062d590cdcd8a96d3b6
```

Comment:
```suggestion
  InstallerSha256: b410e4fbd8602d46df54c77644209887c9f6bf87df08f062d590cdcd8a96d3b6
  ProductCode: '{22F9BFD1-9750-4AAE-8E84-6A3F6401AB34}'
```

Reply:
@mdanish-kh Thank you! update pull request.